Signal/src/util/OWSBackup.h

```diff
@@ -21,16 +24,46 @@ typedef NS_ENUM(NSUInteger, OWSBackupState) {
 
 @interface OWSBackup : NSObject
 
-@property (nonatomic, readonly) OWSBackupState backupExportState;
-
 - (instancetype)init NS_UNAVAILABLE;
 
 + (instancetype)sharedManager;
 
+- (void)setup;
+
+#pragma mark - Backup Export
+
+@property (nonatomic, readonly) OWSBackupState backupExportState;
```

Comment:
I'm still iterating on this, but I might combine the "backup import" and "backup export" properties.